Configurations.md

```diff
@@ -334,16 +334,16 @@ let range = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 #### `"Back"`:
 
 ```rust
-let or = foo ||
-    bar ||
-    foobar;
+fn main() {
+    r = foofoofoofoofoofoofoofoofoofoofoofoofoofoofoofoo ||
+        barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar;
 
-let sum = 1234 +
-    5678 +
-    910;
+    let sum = 123456789012345678901234567890 + 123456789012345678901234567890 +
+        123456789012345678901234567890;
 
-let range = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa..
-    bbbbbbbbbbbbbbbbbbbbbbbbbbbbb;
+    let range = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+        ..bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb;
```

Comment:
Here's the part in question. The previous snipped had `..` at the end of line. Currently, rustfmt with `binop_separator="Back"` places it on the next line. Is this expected?

Reply:
No, this is a bug, I think, `..` should respect the `binop_separator` option

Reply:
:+1:

Here's my thinking on this: I'd like to keep moving forward on the configuration snippets and create issues for bugs identified along the way. For this issue I've created #2364. There's three clear options:
1. Leave the range notation line in this PR and add a comment pointing at #2364. That's nice because when #2364 is fixed, the configuration snippet will break, guaranteeing that it will get updated. The downside is that maybe you don't want bugs noted in the snippets.
1. I remove the line in question from the snippet and make a note in #2364 saying it should be added back before that issue is resolved.
1. We could decide that finishing the configuration snippet changes before dealing with bugs identified isn't the way to go and that we should fix the bug before moving on. (It's also possible someone else will just fix it quickly.)

I'm going to update the PR to reflect Option 1 above. If you prefer a different option, I'll change it.

Reply:
yeah, option 1 sounds good!

Reply:
I would use the format `// FIXME(#1234) a short description of the bug` for the comments. I'll merge this as is, but if you update it in a future PR, that would be good.